release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2313,6 +2366,9 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=18786
 
 * Previously, the topology URLs created for deployments using Bitbucket repository in the {product-title} web console did not work if they included a branch name that contained a slash character. This was due to an issue with the Bitbucket API (link:https://jira.atlassian.com/browse/BCLOUD-9969[BCLOUD-9969]). The current release mitigates this issue. If a branch name contains a slash, the topology URLs point to the default branch page for the repository. This issue will be fixed in a future release of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[*BZ#1970470*].
 
+* Installing {product-title} (OCP) version 4.6 on {rh-virtualization-first} requires RHV version 4.4. If you are running an earlier version of OCP on RHV 4.3, do not update it to OCP version 4.6. Red Hat has not tested running OCP version 4.6 on RHV version 4.3 and does not support this combination. For additional information about tested integrations, see https://access.redhat.com/articles/4763741
+
```

Comment:
Link formatting missing: 

see link:https://access.redhat.com/articles/4763741[OpenShift Container Platform 4.x Tested Integrations (for x86_x64)].